src/razor/src/razorLanguageServerClient.ts

```diff
@@ -250,6 +255,15 @@ export class RazorLanguageServerClient implements vscode.Disposable {
             args.push('true');
             args.push('--UpdateBuffersForClosedDocuments');
             args.push('true');
+
+            if (this.isCSharpDevKitInstalled) {
+                args.push('--telemetryLevel', this.vscodeTelemetryReporter.telemetryLevel);
+                args.push('--sessionId', getSessionId());
+                args.push(
+                    '--extension',
```

Comment:
Super nit, but is the intention for this to be used for other things in future? If not, consider giving it a more specific name.

Reply:
Updated!